packages/hardhat-plugin/src/index.ts

```diff
@@ -125,7 +123,7 @@ task("deploy")
 
       await hre.run("compile", { quiet: true });
 
-      const userModule: Module<ModuleDict> | undefined = loadModule(
+      const userModule: any | undefined = loadModule(
```

Comment:
Why the `any`s in this file?

Reply:
The module loading code needs cleaned up. It hasn't been touched, it just _happens_ to work.

I intend to do this along with the config refactor.